content/docs/user-guide/project-structure/pipelines-files.md

```diff
@@ -114,6 +114,35 @@ stages:
 The commands in `dvc metrics` and `dvc plots` help you display and compare
 metrics and plots.
 
+## Plots definitions
+
+The list of plots contains one or more user-defined
+[plots](/doc/command-reference/plots#plots-definitions). Here's an example that
+tells DVC that `auc.json` is viable for visualization:
+
+```yaml
+stages:
+  build:
+    cmd: python train.py
+    deps:
+      - features.csv
+    outs:
+      - model.pt
+      - auc.json
+    metrics:
+      - accuracy.txt:
+          cache: false
+plots:
+  auc.json:
+    x: fpr
+    y: tpr
+```
+
+Note that in this example we don't have to specify `auc.json` as a plot output.
+Plots definitions accept any file from repository. For more examples how to
+create plots definitions refer to the
+[examples](/doc/command-reference/plots#example-simple-plot-definition).
```

Comment:
We might need more explanation of the syntax either here or in the cmd ref. For example (I think these are correct?):

* The top-level key can be either the path to a file to be used as the data source, or an arbitrary unique name. It will be used as the plot title unless another title is set.
* `y` can have multiple entries to plot multiple data series. Each `y` entry can also be a key-value pair of the file path and field name.

Alternatively, this might fit as comments within the dvc.yaml examples, but it might help to explain a bit even if we want to focus on examples.